tests/ctst/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG SORBET_TAG
 ARG CTST_TAG
 FROM registry.scality.com/sorbet/sorbet:$SORBET_TAG as sorbet
-FROM registry.scality.com/cli-testing/cli-testing:$CTST_TAG
+FROM registry.scality.com/cli-testing-dev/cli-testing:$CTST_TAG
```

Comment:
leftover from local testing ?

Reply:
It's still wip from a dev branch of ctst.